### ISSUE TITLE
SELECT specific columns?

### ISSUE THREAD

**Issue by johnrees** [Casecommons/pg_search#60]
```ruby
# Foo.rb
pg_search_scope :search, against: :name

# Query
Foo.select('id,name').search('bar')
# or
Foo.select('foos.id,foos.name').search('bar')
# or
Foo.search('bar').select('id,name')

# Yields
SELECT id,name, "foos".*...
```
is there a way of avoiding this behaviour and just loading the fields specified without the added select-all directive? As I want to utilise PG9.2's [index-only scans](http://wiki.postgresql.org/wiki/What's_new_in_PostgreSQL_9.2#Index-only_scans) if possible.

**Comment by nertzy** [comment]
Fixed by eb730f705c5c532a7e26b496392b7e09e0b8d713 which has just been added to master.

Test added in a074acbd99da94699be061bf796b6bcb23202a8a.

**Comment by ajb** [comment]
Will test this out in our application shortly. Thanks Grant!
